src/main/java/com/synopsys/integration/alert/channel/msteams/MsTeamsMessage.java

```diff
@@ -87,4 +127,9 @@ public int getProviderCount() {
         return sections;
     }
 
+    //TODO determine if we should be including the URLs
+    private String createLinkableItemString(LinkableItem item) {
```

Comment:
I believe if the item has a url we will want to add it.  Right now it appears that MSTeams doesn't have any URL links.  We would need to investigate the markup for that. Maybe a convenience method on LinkableItem hasUrl which returns a boolean value if the value of getUrl is present.